app/src/main/java/com/android/halanvendordiscovery/domain/details/model/MerchantModel.kt

```diff
@@ -0,0 +1,12 @@
+package com.android.halanvendordiscovery.domain.details.model
+
+data class MerchantModel(
+    val name            : String?   = null,
+    val arabicName      : String?   = null,
+    val address         : String?   = null,
+    val longitude       : Double?   = null,
+    val latitude        : Double?   = null,
+    val phoneNumber     : String?   = null,
+    val vendorName      : String?   = null,
+    val vendorArabicName: String?   = null,
+)
```

Comment:
is all data nullable from backend side?

Reply:
Thanks Tag, I'm working on refactoring it